ts/Dashboards/EditMode/EditMode.ts

```diff
@@ -1082,29 +1091,73 @@ namespace EditMode {
     }
 
     /**
-    * @internal
+    * Tools options.
     */
     export interface Tools {
+        /**
+        * Add Component button options.
+        */
+        addComponentBtn?: AddComponentBtn;
+        /**
+         * Whether the RWD buttons should be visible.
+         *
+         * @default true
+         *
+         */
+        enableRwdButtons?: boolean;
```

Comment:
As I think about it now, instead of `enableRwdButtons`  I would add an interface and option like 
`rwdButtons: RwdButtons` and inside place the `enabled` property and the `icons` as it is now with `addComponentBtn` 

Pseudo code:
```ts
RwdButtons {
   enabled?: boolean,
   icons: {
       ...
   }
}
```

It will be easier to scale this in the future if there is such a need.

Reply:
Right, I didn't want to make changes to an already existing options in API, but I agree with that. I'll fix it.